src/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,4610 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
**question:** Is this file redundant? We're using lang-code `en-us` instead of `en`. I guess this translation file is not being used.